examples/DFWRBE1001Template/teleop.cpp

```diff
@@ -0,0 +1,10 @@
+#include <DFW.h> // DFW include
+
+void teleop( long time,DFW & dfw) { // function definition
+	Serial.print("\r\nTeleop time remaining: ");
+	Serial.print(time);
+	Serial.print(" right joystick: ");
+	Serial.print(dfw.joystickrv());
+	Serial.print(" left joystick: ");

```

Comment:
Instead of prepending the string with a space, do we want to use `\t`?  That way the spacing will be more consistent with changing numbers.